tests/kafkatest/services/performance/producer_performance.py

```diff
@@ -62,6 +63,7 @@ def parse_stats(line):
             }
         last = None
         for line in node.account.ssh_capture(cmd):
+            self.maybe_start_jmx_tool(idx, node)
```

Comment:
@ewencp In response to your question, yeah that is what I have in mind when I was thinking about the test, and is the reason I wanted to have support for "asynchronous" ssh command from Ducktape. But I didn't stated it very clearly in the email, partly because something become clearer only after I started implementation -- for example, I once thought that adding "&" would solve problem cleanly but it didn't.